TradeItIosEmsApi/TradeItConnector.m

```diff
@@ -72,124 +70,30 @@ - (id)initWithApiKey:(NSString *)apiKey
-                                                                        emsAction:@"user/getOAuthAccessToken"
-                                                                      environment:self.environment];
-
-    [self sendEMSRequest:request
-     withCompletionBlock:^(TradeItResult *tradeItResult, NSMutableString *jsonResponse) {
-         if ([tradeItResult isSuccessful]) {
-             TradeItOAuthAccessTokenResult *successResult
-             = (TradeItOAuthAccessTokenResult *)[TradeItRequestResultFactory buildResult:[TradeItOAuthAccessTokenResult alloc]
-                                                                              jsonString:jsonResponse];
-             tradeItResult = successResult;
-         }
-
-         completionBlock(tradeItResult);
-     }];
-}
+- (id)initWithApiKey:(NSString *)apiKey {
+    self = [super init];
 
-- (void)getAvailableBrokersWithCompletionBlock:(void (^ _Nullable)(NSArray<TradeItBroker *> * _Nullable, NSString * _Nullable))completionBlock {
-    [self getAvailableBrokersWithUserCountryCode:nil
-                                 completionBlock:completionBlock];
-}
+    if (self) {
+        self.apiKey = apiKey;
+        self.environment = TradeItEmsProductionEnv;
+        self.version = TradeItEmsApiVersion_2;
+        runAsyncCompletionBlockOnMainThread = true;
+    }
 
-- (void)getAvailableBrokersWithUserCountryCode:(NSString * _Nullable)userCountryCode
-                               completionBlock:(void (^ _Nullable)(NSArray<TradeItBroker *> * _Nullable, NSString * _Nullable))completionBlock
-
-{
-    TradeItBrokerListRequest *brokerListRequest = [[TradeItBrokerListRequest alloc] initWithApiKey:self.apiKey
-                                                                                   userCountryCode:userCountryCode];
-
-    NSURLRequest *request = [TradeItRequestResultFactory buildJsonRequestForModel:brokerListRequest
-                                                                        emsAction:@"preference/getBrokerList"
-                                                                      environment:self.environment];
-
-    [self sendEMSRequest:request forResultClass:[TradeItBrokerListResult class] withCompletionBlock:^(TradeItResult *result) {
-        if ([result isKindOfClass: [TradeItBrokerListResult class]]) {
-            TradeItBrokerListResult *brokerListResult = (TradeItBrokerListResult *)result;
-//            NSLog(@"\n\n\n=====> brokerListResult: %@\n\n\n", brokerListResult);
-            completionBlock(brokerListResult.brokerList, brokerListResult.featuredBrokerLabel);
-        } else {
-            NSLog(@"Could not fetch broker list; got error result: %@", result);
-            completionBlock(nil, nil);
-        }
-    }];
+    return self;
 }
 
-
+// TODO: Deprecated - should be using OAuth
 - (void)linkBrokerWithAuthenticationInfo:(TradeItAuthenticationInfo *)authInfo
```

Comment:
It's marked as internal so no one should be able to use it. Let's get rid of it.